### DISCUSSION TITLE
Could you please tell me how to test site-to-site case?

### DISCUSSION THREAD

**Discussion by zbb88888** [strongswan/strongswan#1729] [Q&A] [answered]
Is there any doc about how to test site-to-site case?

**Comment by SvenRoederer** [comment]
Have you taken a look at https://www.strongswan.org/testing/testresults/ikev2/index.html, esp the "net2net" and "host"host" examples. They have been a good starting point to me.

**Reply by zbb88888** [reply, marked as answer]
Thank you， i will give it a look
